sequence_processing_pipeline/Pipeline.py

```diff
@@ -276,6 +297,55 @@ def _validate_sample_sheet(self, sample_sheet_path):
                               isinstance(x, WarningMessage)]
             return val_sheet
 
+    def _validate_mapping_file(self, mapping_file_path):
+        """
+        Performs validation for mapping-files.
+        :return: If successful, an empty list of strings and a valid
+                 mapping-file. If unsuccessful, a list of warning and error
+                 messages and None.
+        """
+        try:
+            df = pd.read_csv(mapping_file_path, delimiter='\t', dtype=str)
+        except pd.errors.ParserError:
+            # ignore parser errors as they obviously prove this is not a
+            # valid mapping file. Return None w/error message
```

Comment:
This is not actually true, right? It still raises an error.

Reply:
Thanks for the catch! The behavior is analogous to the existing _validate_sample_sheet(), but the comments and doctoring for both needed to be updated.

Reply:
... so, not sure I follow, are you planning to fix the message or change the code?